policy-controller/k8s/index/src/inbound/index.rs

```diff
@@ -174,6 +174,18 @@ struct NsUpdate<K, T> {
     removed: HashSet<K>,
 }
 
+trait DefaultInboundRoutes<MatchType> {
+    fn default_inbound_routes<'p>(
```

Comment:
The default routes (when no HttpRoute or GrpcRoute resources) will always be HTTP (probe routes).  Therefore there's no reason to make this generic or have a trait for it.

Reply:
Reverted